api/specs/webserver/scripts/openapi_wallets.py

```diff
@@ -0,0 +1,119 @@
+""" Helper script to generate OAS automatically
+"""
+
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+
+from enum import Enum
+
+from fastapi import FastAPI, status
+from models_library.api_schemas_webserver.wallets import WalletGet
+from models_library.generics import Envelope
+from models_library.users import GroupID
+from models_library.wallets import WalletID
+from simcore_service_webserver.wallets._groups_api import WalletGroupGet
+from simcore_service_webserver.wallets._groups_handlers import _WalletsGroupsBodyParams
+
+# from simcore_service_webserver.tags._handlers import TagCreate, TagGet, TagUpdate
+from simcore_service_webserver.wallets._handlers import (
+    _CreateWalletBodyParams,
+    _PutWalletBodyParams,
+)
+
+app = FastAPI(redoc_url=None)
+
+TAGS: list[str | Enum] = [
+    "wallets",
+]
+
+
+### Wallets
+
+
+@app.post(
+    "/wallets",
+    response_model=Envelope[WalletGet],
+    tags=TAGS,
+    operation_id="create_wallet",
+    status_code=201,
+)
+async def create_wallet(body: _CreateWalletBodyParams):
+    ...
+
+
+@app.get(
+    "/wallets",
+    response_model=Envelope[list[WalletGet]],
+    tags=TAGS,
+    operation_id="list_wallets",
+)
+async def list_wallets():
+    ...
+
+
+@app.put(
+    "/wallets/{wallet_id}",
+    response_model=Envelope[WalletGet],
+    tags=TAGS,
+    operation_id="update_wallet",
+)
+async def update_wallet(wallet_id: WalletID, body: _PutWalletBodyParams):
```

Comment:
Question: PUT is for replacing the wallet. is there no need to PATCH only the contents? that is just a thought for later.

Reply:
yes PATCH can be added later if needed :+1: